### PR TITLE
feat: add `additionalContainerClasses`

### PR DESCRIPTION
**Pull request by evolkmann**

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -61,6 +71,21 @@ export class MapboxInfoBoxControl implements IControl
...
 
+    private createContainer(options: IMapboxInfoBoxOptions) {
+        this.controlContainer = document.createElement("div");
+        this.controlContainer.classList.add("mapboxgl-ctrl");
+        this.controlContainer.classList.add("mapboxgl-ctrl-group");
+        this.controlContainer.classList.add("mapboxgl-ctrl-icon");
+        this.controlContainer.classList.add("mapboxgl-info-box-ctrl");
+
+        if (options.additionalContainerClasses?.length) {
+            const classes = Array.isArray(options.additionalContainerClasses) ? options.additionalContainerClasses : [options.additionalContainerClasses];
```
Comment (el) [inline lib/index.ts]: If you change it to this, we can support multiple class names as a string
```
            const classes = Array.isArray(options.additionalContainerClasses) ? options.additionalContainerClasses : options.additionalContainerClasses.split(" ");
```

File: lib/index.ts [hunk 2]
```diff
@@ -61,6 +71,21 @@ export class MapboxInfoBoxControl implements IControl
...
     }
 
+    private createContainer(options: IMapboxInfoBoxOptions) {
+        this.controlContainer = document.createElement("div");
+        this.controlContainer.classList.add("mapboxgl-ctrl");
+        this.controlContainer.classList.add("mapboxgl-ctrl-group");
+        this.controlContainer.classList.add("mapboxgl-ctrl-icon");
+        this.controlContainer.classList.add("mapboxgl-info-box-ctrl");
+
+        if (options.additionalContainerClasses?.length) {
```
Comment (el) [inline lib/index.ts:81]: additionalContainerClasses might be a string

```
        if (options.additionalContainerClasses)
        {
```

File: lib/index.ts [hunk 3]
```diff
@@ -61,6 +71,21 @@ export class MapboxInfoBoxControl implements IControl
         this.map.off("mousemove", this.layerId, this.handleMouseMove);
     }
 
+    private createContainer(options: IMapboxInfoBoxOptions) {
```
Comment (el) [inline lib/index.ts]:
```
    private createContainer(options: IMapboxInfoBoxOptions): void
    {

```

File: lib/index.ts [hunk 4]
```diff
@@ -6,6 +6,20 @@ export interface IMapboxInfoBoxOptions
 {
     layerId?: string,
     formatter?: (properties: GeoJsonProperties) => string
+    /**
```
Comment (el) [inline lib/index.ts]: unnecessary comments

File: lib/index.ts [hunk 5]
```diff
@@ -61,6 +71,21 @@ export class MapboxInfoBoxControl implements IControl
...
+    private createContainer(options: IMapboxInfoBoxOptions) {
+        this.controlContainer = document.createElement("div");
+        this.controlContainer.classList.add("mapboxgl-ctrl");
+        this.controlContainer.classList.add("mapboxgl-ctrl-group");
+        this.controlContainer.classList.add("mapboxgl-ctrl-icon");
+        this.controlContainer.classList.add("mapboxgl-info-box-ctrl");
+
+        if (options.additionalContainerClasses?.length) {
+            const classes = Array.isArray(options.additionalContainerClasses) ? options.additionalContainerClasses : [options.additionalContainerClasses];
+            for (const className of classes) {
```
Comment (el) [inline lib/index.ts]:
```
this.controlContainer.classList.add(...classes);
```

**Review by el — CHANGES_REQUESTED** [review]
Thank you for the PR, if you can make the changes requested, I can take another look :+1:

**Comment by evolkmann** [comment]
Hi @el, thanks for your comments; I agree 👍 
I have made the changes.

**Pull request merged**